### DISCUSSION TITLE
DA_CAPSTONE | ACCESS URL LINK : TypeError: can only concatenate str (not "int") to str

### DISCUSSION THREAD

**Discussion by FaizFaisalAhmad** [teamalgoritma/community#663] [Q&A] [answered]
I have weird error,

in VS code, run python app.py , successfully without any Error

![image](https://github.com/teamalgoritma/community/assets/81740393/33dabbb0-dc66-4d4b-abef-7a8384c87d36)

then when i try to access link, then suddenly error appears

![image](https://github.com/teamalgoritma/community/assets/81740393/4e73ebfa-858e-4c4b-a743-0345d87473f4)

![image](https://github.com/teamalgoritma/community/assets/81740393/2a1fc1b3-826c-47d5-a270-ec8f5257ef8f)

**Comment by nadyadtm** [comment]
Dear @FaizFaisalAhmad 

Apakah bisa dilihat untuk code dari apps.py nya?

**Reply by FaizFaisalAhmad** [reply]
> Dear @FaizFaisalAhmad
> 
> Apakah bisa dilihat untuk code dari apps.py nya?


ini Code nya
[app.zip](https://github.com/teamalgoritma/community/files/11473668/app.zip)

**Reply by nadyadtm** [reply]
Baik pak @FaizFaisalAhmad kami sedang mengecek file tersebut. Akan kami infokan bila sudah selesai. Terima Kasih

**Reply by nadyadtm** [reply, marked as answer]
Dear Pak @FaizFaisalAhmad 

Kesalahannya ada di bagian `df["PostedDay"].mean().round(2)`. Karena dataframe df sepertinya merupakan dataframe yang belum di preprocessing (seluruh tipe datanya masih object/string) jadi silakan untuk menyesuaikannya kembali (menggunakan df_merged). Lalu untuk bagian ini `ax = df.plot(figsize = (20,9))` silakan disesuaikan lagi dengan menggunakan dataframe tersebut.

Semoga membantu

**Reply by FaizFaisalAhmad** [reply]
terima kasih atas bantuannya mbak nadya, sudah bisa skrg

**Reply by nadyadtm** [reply]
Baik pak, sama-sama
